core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -211,6 +211,7 @@
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOG2;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOGICAL_AND;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOGICAL_OR;
+import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOG_MS;
```

Comment:
We really should look into some other mechanism for operators implemented differently across different dialects. I feel that just modifying the name is confusing and not very sustainable (what if 2 dialects support 1 version and 2 support another? what do you name it then?) I know it's not in the scope of this change but just putting it out there.

Reply:
I think we could rethink how we use the @LibraryAnnotations because it seems like we have a straightforward way of referencing what dialects a given operator supports.

Reply:
@tanclary I very much agree with your point of view, I am also thinking about this issue, maybe we should take the first step here with the log function. Do you think I should send an email to discuss it?